sdk-workflows/src/main/java/io/dapr/workflows/DaprWorkflowContextImpl.java

```diff
@@ -51,6 +53,23 @@ public DaprWorkflowContextImpl(TaskOrchestrationContext context) throws IllegalA
    * @throws IllegalArgumentException if context or logger is null
    */
   public DaprWorkflowContextImpl(TaskOrchestrationContext context, Logger logger) throws IllegalArgumentException {
+    this(context, logger, null);
+  }
+
+  public DaprWorkflowContextImpl(TaskOrchestrationContext context, Saga saga) throws IllegalArgumentException {
```

Comment:
Can the Saga pattern be separate classes that takes in the workflow as params in the constructor? In this case, it would be DaprWorkflowSagaContextImpl.

cc @cgillum

Reply:
PR code updated. Add SagaContext and getSagaContext() in WorkflowContext.